<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Select_form</class>
 <widget class="QDialog" name="Select_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="ScenarioList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>256</width>
     <height>231</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ScenarioBrief">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>256</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="FractionList">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>21</y>
     <width>211</width>
     <height>301</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Start">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Start</sender>
   <signal>clicked()</signal>
   <receiver>Select_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Select_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
